yb-voyager/src/pgss/stats.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package pgss
+
+// QueryStats represents a pg_stat_statements entry
+// All field names follow PostgreSQL 13+ conventions for consistency across versions
+// This struct serves as the single source of truth for PGSS data throughout the system
+type QueryStats struct {
```

Comment:
Shall we just name this `PgStatStatement` so that it's clear from the name itself what it is?

Reply:
I was thinking about this again, let's discuss this on call if required. 

I think 
- `pg_stat_statements` should only be relevant in the context of fetching/parsing (it is a pg/yb specific detail), no other place. 
- AssessmentDB/compare-perf packages should not have any exposure to `pg_stat_statements`; they should instead deal with a `QueryStats` abstraction.


As a result, we should probably define this struct somewhere else then.

Reply:
If it helps i can define a PgStatStatements separately and inherit that here(go struct embedding)